src/main/java/com/insert/ogbsm/domain/timeTable/Period.java

```diff
@@ -0,0 +1,264 @@
+package com.insert.ogbsm.domain.timeTable;
+
+import com.insert.ogbsm.infra.error.exception.BsmException;
+import com.insert.ogbsm.infra.error.exception.ErrorCode;
+import com.insert.ogbsm.presentation.timeTable.dto.TimeTableValueRes;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.time.DayOfWeek;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static java.time.DayOfWeek.*;
+
+@AllArgsConstructor
+@Getter
+public enum Period {
+    // 기숙사
+    MORNING_DORM(new Time(0, 0), new Time(6, 30)),
+    MORNING_DORM_FRIDAY(new Time(0, 0), new Time(7, 30)),
+    PREPARING_DORM(new Time(6, 50), new Time(7, 30)),
+    NIGHT_DORM(new Time(20, 40), new Time(24, 0)),
+
+    // 아침운동
+    MORNING_EXERSIZE(new Time(6, 30), new Time(6, 50)),
+
+    //아침 자습시간
+    MORNING_SELF_STUDY(new Time(8, 5), new Time(8, 30)),
+
+    //시간표
+    FIRST(new Time(8, 40), new Time(9, 30)),
+    SECOND(new Time(9, 40), new Time(10, 30)),
+    THIRD(new Time(10, 40), new Time(11, 30)),
+    FOURTH(new Time(11, 40), new Time(12, 30)),
+    FIFTH(new Time(1, 20), new Time(2, 10)),
+    SIXTH(new Time(2, 20), new Time(3, 10)),
+    SEVENTH(new Time(3, 20), new Time(4, 10)),
+    AFTER_SCHOOL_1(new Time(4, 30), new Time(6, 10)),
+    AFTER_SCHOOL_2(new Time(7, 0), new Time(8, 40)),
+
+    //조례, 종례
+    PRONUNCIATION(new Time(8, 30), new Time(8, 40)),
+    ANTONYM(new Time(4, 20), new Time(4, 30)),
+    CLEANING(new Time(4, 10), new Time(4, 20)),
+
+    //쉬는 시간
+    BREAK_1(new Time(9, 30), new Time(9, 40)),
+    BREAK_2(new Time(10, 30), new Time(10, 40)),
+    BREAK_3(new Time(11, 30), new Time(11, 40)),
+    BREAK_4(new Time(2, 10), new Time(2, 20)),
+    BREAK_5(new Time(3, 10), new Time(3, 20)),
+
+    //식사
+    BREAKFAST(new Time(7, 30), new Time(8, 5)),
+    LUNCH(new Time(12, 30), new Time(1, 20)),
+    DINNER(new Time(6, 10), new Time(7, 0)),
+
+    //금요일
+    ANTONYM_FRIDAY(new Time(3, 10), new Time(3, 20)),
+    GO_HOME(new Time(3, 20), new Time(24, 0)),
+
+    //토요일
+    HOME(new Time(0, 0), new Time(24, 0)),
+
+    //일요일
+    HOME_SUNDAY(new Time(0, 0), new Time(7, 50)),
+    DORM_SUNDAY(new Time(7, 50), new Time(24, 0));
+
+
+    final Time startTime;
+    final Time endTime;
+
+    public static Period get(int period) {
+        switch (period) {
+            case 1:
+                return FIRST;
+            case 2:
+                return SECOND;
+            case 3:
+                return THIRD;
+            case 4:
+                return FOURTH;
+            case 5:
+                return FIFTH;
+            case 6:
+                return SIXTH;
+            case 7:
+                return SEVENTH;
+        }
+        throw new BsmException(ErrorCode.NO_PERIOD_MATCHED);
```

Comment:
default를 사용하고 switch자채를 리턴시킬수 있어요
```suggestion
      return switch (period) {
            case 1 -> FIRST;
            case 2 -> SECOND;
            ...
            default -> throw new BsmException(ErrorCode.NO_PERIOD_MATCHED);
        }
     
```

Reply:
<img width="318" alt="스크린샷 2023-10-03 오후 3 01 20" src="https://github.com/Team-INSERT/Something_Big_Backend/assets/101192100/67247103-6086-4fae-a343-3d8bd762ec94">

혹시 제가 잘못 이해한건가요?? ㅠㅠㅠ